src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.cpp

```diff
@@ -49,6 +49,8 @@ void CARMAStreetsPlugin::UpdateConfigSettings() {
  	GetConfigValue<string>("MapTopic", _transmitMAPTopic);
 	GetConfigValue<string>("SRMTopic", _transmitSRMTopic); 
 	GetConfigValue<string>("SimSensorDetectedObjTopic", _transmitSimSensorDetectedObjTopic); 
+	GetConfigValue<string>("SdsmTopic", _subscribeToSdsmTopic);
+	GetConfigValue<string>("SdsmTopic", _transmitSDSMTopic);
```

Comment:
Same here

Reply:
Yep figured I missed something minor. I meant to ask- can/should these use the same name? The other configs seem to use [x]Topic interchangeably for subscribe/transmit but this one does both